scripts/pico_project.py

```diff
@@ -577,7 +577,10 @@ def GenerateCMake(folder, params):
                 f"set(sdkVersion {params['sdkVersion']})\n"
                 f"set(toolchainVersion {params['toolchainVersion']})\n"
                 f"set(picotoolVersion {params['picotoolVersion']})\n"
-                f"include({cmakeIncPath()})\n"
+                f"set(picoVscode {cmakeIncPath()})\n"
+                "if (EXISTS ${picoVscode})\n"
+                "    include(${picoVscode})\n"
```

Comment:
@will-v-pi it would be nice to set the `PICO_PLATFORM` in the main `CMakeLists.txt` too as it allows projects that use riscv to be shared with others that do not have the extension installed.

Reply:
I think I'd prefer to stick to defaulting the pico platform based off the board (as the SDK does), and only changing it to Risc-V when using a Risc-V toolchain. For projects that require Risc-V and may have users not using the extension, they can add this to their CMakeLists.txt file to set the platform
```
if (NOT DEFINED PICO_PLATFORM)
    set(PICO_PLATFORM rp2350-riscv CACHE STRING "Pico Platform")
endif()
```